### PR TITLE
Modifications for SSL

### PR DESCRIPTION
**Pull request by dominicschaff**

### CODE REVIEW

File: setup-pg_hba.sh [hunk 1]
```diff
@@ -2,16 +2,40 @@
...
 
-# Custom IP range via docker run -e (https://docs.docker.com/engine/reference/run/#env-environment-variables)
-# Usage is: docker run [...] -e ALLOW_IP_RANGE='192.168.0.0/16'
+# Reconfigure pg_hba if environment settings changed
+# cat ${ROOT_CONF}/pg_hba.conf.template > ${ROOT_CONF}/pg_hba.conf
+
+
+if [[ "${FORCE_SSL}" =~ [Ff][Aa][Ll][Ss][Ee] ]]; then
+  PG_CONF_HOST='host'
+  CERT_AUTH=${PASSWORD_AUTHENTICATION}
```
Comment (NyakudyaA) [inline setup-pg_hba.sh:18]: Is the PASSWORD_AUTH defined anywhere in the env-data

Comment (dominicschaff) [reply setup-pg_hba.sh:18]: Not that I can find. I shall copy and test again

Comment (dominicschaff) [reply setup-pg_hba.sh:18]: LOG:  invalid authentication method "scram-sha-256"

Comment (dominicschaff) [reply setup-pg_hba.sh:18]: Switched to md5 now it is running

Comment (NyakudyaA) [reply setup-pg_hba.sh:18]: scram-sha-256 is only available in PostgreSQL 10 and above

**Comment by NyakudyaA** [comment]
LGTM - There are other changes that we can add to the image that is coming from the 13 images but I can do a PR after you have merged here as a followup

**Pull request merged**